pyproject.toml

```diff
@@ -146,25 +146,22 @@ ECM_Example = "pybamm.input.parameters.ecm.example_set:get_parameter_values"
 MSMR_Example = "pybamm.input.parameters.lithium_ion.MSMR_example_set:get_parameter_values"
 Chayambuka2022 = "pybamm.input.parameters.sodium_ion.Chayambuka2022:get_parameter_values"
 
-[tool.setuptools]
-include-package-data = true
+[tool.hatch.build]
+packages = ["src/pybamm"]
+include = [
+    "src/pybamm/**/*.txt",
+    "src/pybamm/**/*.md",
+    "src/pybamm/**/*.csv",
+    "src/pybamm/**/*.py",
+    "src/pybamm/CITATIONS.bib",
+    "src/pybamm/plotting/mplstyle",
```

Comment:
I don't see this file in the repo

Reply:
See [my comment below](https://github.com/pybamm-team/PyBaMM/pull/4747#discussion_r1913857022).